simulateCoOp.py

```diff
@@ -0,0 +1,176 @@
+import farmRe as farmRe
+import functions as functions
+
+#importlib.reload(farm)
+
+def compileCoOp(farmStr): # strategyStr = None, treeStr = None):
+    """
+    Takes string arguments (file paths) and compiles the data into a list of classes of type : Cuerdas, assigning the parameters respective to the data in the spreadsheet (farmer names, tree types, number of cuerdas, and age of trees).
+    
+    Parameters
+    ----------
+    farmStr : str
+        string of filepath (from current working directory) to spreadsheet with data. Spreadsheet must correspond with template to run function. 
+        
+        
+    Returns
+    -------
+    lsOfPlots : list of farm.Cuerdas
+        a list of class type Cuerdas that have been initialized with their respective parameters and attributes.
+    
+    """
+    farmData = functions.readData(farmStr)
+
+    lsOfPlots = []
+
+    # create a list of plots using class Cuerdas
+    for i in range(len(farmData)):
+        tempName = str(farmData['farmerName'][i])
+        tempCuerdas = float(farmData['numCuerdas'][i])
+        tempTree = str(farmData['treeType'][i])
+        tempAge = float(farmData['ageOfTrees'][i])
+    
+        plot = farmRe.Farm(_farmerName=tempName, _cuerdas=tempCuerdas, _treeType=tempTree, _initialAgeOfTrees=tempAge)
+        
+        lsOfPlots.append(plot)
+        
+    return(lsOfPlots)
+
+
+def simulateCoOp(lsOfPlots, numYears, pruneYear = None, growthPattern = None, strategy = None):
+
+    numPlots = len(lsOfPlots)
+
+    annualHarvest = []
+    harvestYear = []
+    
+    
+
+    for year in range(numYears):
+        thisYearsHarvest = 0 # each year reset harvest
+
+        for j in range(numPlots):
+            if (pruneYear):
+                if j == pruneYear: # if it's the prune year
+                    # isPrune = True
+                    lsOfPlots[j].setPruneTrees()
+                    
+            lsOfPlots[j].oneYear() # run this plot through one year of the demo
+            tempHarvest = lsOfPlots[j].totalHarvest
+            lsOfPlots[j].setHarvestZero() # not cumulative sum, but instead reset
+            thisYearsHarvest += tempHarvest
+
+        harvestYear.append(year)
+        annualHarvest.append(thisYearsHarvest)
+       
+        
+    simulation = [harvestYear, annualHarvest]
+    
+    return(simulation)
+
+
+        
+
+
+def main():
+    import argparse
+    import os
+    parser = argparse.ArgumentParser(description='Parse growth data for simulation.')
+    parser.add_argument('-f', '--farm',
+                        default='data/demoData.csv',
+                        type=str, # string type works well for 
+                        help=
+                        """
+                        Name of file (and path from current directory) to data containing cuerdas, tree types, etc.
+                        
+                        Example (& default): --farm data/demoData.csv
+                        
+                        """)
+    
```

Comment:
argparse in separate method from main.

main already takes in the set of arguments `main(**kwds)`